specs/003-login-ui/quickstart.md

```
@@ -0,0 +1,333 @@
+- No real backend call
+
+### 5. **Route Protection**: Single Auth Guard
+- Check for session token in guard logic
+- Redirect unauthenticated users to `/login`
+- Redirect authenticated users away from `/login` to `/chat`
+
+---
+
+## Architecture Overview
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)